tests/encoder_decoder/test_e2e_correctness.py

```diff
@@ -29,7 +29,7 @@ def vllm_to_hf_output(
 
 
 @pytest.mark.parametrize("model", ["facebook/bart-large-cnn"])
```

Comment:
Should we check both xformer and flash attention works e2e here?

Reply:
Its already doing it because of the dtype being "bfloat16" & "float". I tried adding an assert for the backend type but was not able to do it. I verified from logs it uses both.

Reply:
can we call `global_force_attn_backend` in this function to manually set the attention backend? We can test 3 cases, automatic selection, flashattn, xformer. I think using "bfloat16" & "float" to trigger this is not stable.